integrations/access/msteams/app.go

```diff
@@ -145,6 +150,24 @@ func (a *App) init(ctx context.Context) error {
 		return trace.Wrap(err)
 	}
 
+	a.accessMonitoringRules = accessmonitoring.NewRuleHandler(accessmonitoring.RuleHandlerConfig{
```

Comment:
Are we missing the PluginType?
It doesn't seem to be used, but other plugins have it.

Reply:
The plugin type is not needed. The rules are now grouped by name instead of type.